test-suite/CMakeLists.txt

```diff
@@ -0,0 +1,129 @@
+cmake_minimum_required(VERSION 3.20...3.29)
+
+#[=============================================================================[
+#                           Basic project definition                           #
+]=============================================================================]
+
+list(APPEND CMAKE_MESSAGE_CONTEXT Test)
+project(Wannier90_test
+		LANGUAGES C Fortran
+)
+
+set(CMAKE_C_STANDARD 11)
+set(CMAKE_C_STANDARD_REQUIRED ON)
+set(CMAKE_C_EXTENSIONS OFF)
+
+#[=============================================================================[
+#                                   Options                                   #
+]=============================================================================]
+
+#[=============================================================================[
+#                            Project configuration                            #
+]=============================================================================]
+
+#[=============================================================================[
+#                              External packages                              #
+]=============================================================================]
+
+find_package(Python 3.8 REQUIRED)
+
+#[=============================================================================[
+#                               Main definition                               #
+]=============================================================================]
+
+## Prepare test environment
+foreach (extra_file IN ITEMS
+		checkpoints
+		run_tests
+		testcode
+		tests
+		tools
+)
+	file(COPY ${extra_file} DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
+endforeach ()
+
+set(test_names
```

Comment:
These tests are out-of-sync @JeromeCCP9, can you help update these?

Reply:
I think I have synced them again, but I am not calling parallelism correctly, I will try to fix that.